package.json

```diff
@@ -87,7 +87,7 @@
     "@hashicorp/react-code-block": "^4.5.0",
     "@hashicorp/react-command-line-terminal": "^2.0.4",
     "@hashicorp/react-consent-manager": "^7.1.0",
-    "@hashicorp/react-docs-page": "^14.14.3",
+    "@hashicorp/react-docs-page": "^14.14.5-canary-20220318185541",
```

Comment:
Assuming this will get bumped before merge (once https://github.com/hashicorp/react-components/pull/541 lands & merges & publishes)